draft-ietf-privacypass-protocol.md

```diff
@@ -230,22 +218,48 @@ clients and servers supporting the Privacy Pass protocol.
   current public parts of the configurations used the server.
 
 The wider ecosystem that this protocol is employed in is described in
-{{draft-davidson-pp-architecture}}.
+{{I-D.ietf-privacypass-architecture}}.
+
+# Notation
+
+The following terms are used throughout this document to describe the
+protocol operations in this document:
+
+- I2OSP and OS2IP: Convert a byte string to and from a non-negative integer as
+  described in {{!RFC8017}}. Note that these functions operate on byte strings
+  in big-endian byte order.
+- random_bytes(L): Generate a random, uniformly distributed byte string of
+  length L.
+- len(s): The length of a byte string, in octets.
 
-# Protocol description {#overview}
+# Protocol description {#pp-protocol}
 
-The Privacy Pass protocol is split into two phases that are built upon
-the functionality described in {{pp-api}} later.
+The Privacy Pass protocol is split into the two following sub-protocols:
 
-The first phase, "issuance", provides the client with unlinkable tokens
+1. Issuance: this protocol provides the client with unlinkable tokens
 that can be used to initiate re-authorization with the server in the
-future. The second phase, "redemption", allows the client to redeem a
-given re-authorization token with the server that it interacted with
-during the issuance phase. The protocol must satisfy two cryptographic
+future.
+2. Redemption: this protocol allows the client to redeem a given
+re-authorization token with the server that it interacted with during
+the Issuance protocol. The protocol must satisfy two cryptographic
 security requirements known as "unlinkability" and "unforgeability".
 These requirements are covered in {{sec-reqs}}.
 
-## Server setup {#server-setup}
+The data structures and protocol messages used throughout the remainder
+of this document are written in the TLS presentation language {{RFC8446, Section 3}}.
+
+## Variants {#variants}
+
+This document specifies a single Privacy Pass variant, defined by the
+following enumeration:
+
+~~~
+enum {
+  VOPRF_decaf448_shake256 = 0x0001,
```

Comment:
Any chance we can add the p384_sha512 variant?

Reply:
Yeah, I was just getting one down to start.